deployment/ib-kubernetes-ufm-secret.yaml

```diff
@@ -1,12 +1,14 @@
 apiVersion: v1
+data:
+  UFM_ADDRESS: ci1kY3M1LTAy # 127.0.0.1 - base64 format
```

Comment:
why is UFM address in base 64 format  and not converted in the plugin ? this parameter is user configurable

Reply:
yes I can move it to stringData